content/blog/2020-05-04-dvc-3-years-and-1-0-release.md

```diff
@@ -0,0 +1,198 @@
+---
+title: DVC 3 Years Anniversary and 1.0 Pre-release
+date: 2020-05-04
+description: |
+  Today, we’ve got three big things to announce: 🎉 3rd-year anniversary of DVC,
+  🚀 DVC 1.0 pre-release is ready and ⭐ 5000 GitHub starts.
+
+descriptionLong: |
+  Today, we’ve got three big things to announce.
+
+  🎉 3rd-year anniversary of DVC
+
+  🚀 DVC 1.0 pre-release is ready
+
+  ⭐ DVC has reached 5K GitHub starts (coincidently on the same day)
+
+  We are sharing our learnings from this journey and how they affected the new
```

Comment:
The last sentence is a bit awkward. Maybe, `We'll share what we've learned from our journey and how DVC is growing.`

Reply:
I'm trying to make a connection between the learning and the features we release. How about `We'll share what we've learned from our journey, how it helped for the release and how DVC is growing.`?